airflow/www/templates/airflow/nvd3.html

```diff
@@ -162,7 +162,7 @@ <h4 class="panel-title">
           $('#chart_body').html('<div class="alert alert-danger">' + payload.error + '</div>');
         }
         $("#sql_panel_body").html(payload.sql_html);
-        $("#label").html(payload.label);
+        $("#label").html(escapeHTML(payload.label));
```

Comment:
```suggestion
        $("#label").text(payload.label);
```

Reply:
This is the current implementation of the escapeHtml function
```
var el = document.createElement("span");

export function escapeHtml(text) {
  el.textContent = text;
  return el.innerHTML;
}
```
``$("#label").html(content)`` is a equivalent of ``document.querySelector("#label").innerHTML = content``
``$("#label").text(content)`` is a equivalent of ``document.querySelector("#label").textContent = content``
Setting textContent is secure and does not cause security problems.

Reply:
Fair point